src/fn/upload-custom-translations.js

```diff
@@ -53,12 +53,16 @@ const execute = async () => {
 
     overwriteProperties(doc, translations);
 
-    await warnUploadOverwrite.preUploadByRev(db, doc);
+    const changes = await warnUploadOverwrite.preUploadDoc(db, doc);
```

Comment:
The function now returns a boolean for whether or not anything has actually changed.